restructuredtext_lint/cli.py

```diff
@@ -12,9 +12,12 @@
     VERSION = version_file.read().strip()
 
 
-def _main(filepaths, format='text', stream=sys.stdout, encoding=None):
+def _main(filepaths, format='text', stream=sys.stdout, encoding=None, report=2):
     error_dicts = []
-    error_occurred = False
+    error_level = 0  # No error
+    file_error_level = 0
+
+    assert 0 <= report <= 4, report
```

Comment:
Since we are using `choices` in `argparse`, we can probably get rid of this assert. If we still want to keep it, then let's provide a better message for developers or remove the message as "10" is quite vague:

```python
assert 0 <= report <= 4
# OR
assert 0 <= report <= 4, 'Expected `report` between 0 and 4 but received "{report}"'.format(report=report)
```